hbase-rest-read-write/cloudera/cod/examples/write_example.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+from requests.auth import HTTPBasicAuth
+from common import *
+import json
+import requests
+from datetime import datetime
+try:
+    from UserDict import UserDict
+    from UserDict import DictMixin
+except ImportError:
+    from collections import UserDict, OrderedDict
+    from collections import MutableMapping as DictMixin
+
+# change: change input for b64encode method from str to bytes
+username_column_encoded = base64.b64encode(bytes(cf_name + ":" + username_column, 'utf-8'))
+message_column_encoded = base64.b64encode(bytes(cf_name + ":" + message_column, 'utf-8'))
+created_time_column_encoded = base64.b64encode(bytes(cf_name + ":" + created_time, 'utf-8'))
+
+# Delete table if it exists
+# Change: Add verify and auth part
+request = requests.get(baseurl + "/" + table_name + "/schema", verify=False,
```

Comment:
Just to remember: verify=False is only required for Dev environment, we will remove this before the final commit.